spec/passenger_events.schema.json

```diff
@@ -31,6 +31,12 @@
         "required": true
       }
     },
+    {
+      "name": "location_ping_id",
+      "type": "string",
+      "title": "ID referencing vehicle_locations.location_ping_id",
+      "description": "Identifies a vehicle_location that is the location where the passenger_event occurred"
```

Comment:
See comment in fare_transactions.schema.json

Reply:
Agreed. Thanks.